src/glossary.adoc

```diff
@@ -0,0 +1,116 @@
+[[sec:glossary]]
+[glossary]
+= Glossary
+
+[glossary]
+
+arity::
+The number of arguments declared by or passed to a macro.  A macro can be _fixed_ or _variable_
+arity; the number of subforms in an invocation is its _actual arity_.
```

Comment:
Is this just the count of all arguments or is this the number of parameters that a macro has?